.github/workflows/integration.yml

```diff
@@ -136,8 +150,21 @@ jobs:
   e2e-test-trigger:
      runs-on: ubuntu-latest
      needs: [build, configure]
-     if: github.event.client_payload.slash_command.command == 'test'
      steps:
+
+       # Since we are using a repository-dispatch event, we have to explicitly set a run check. We initialize it to a "pending" state.
```

Comment:
The comment should be updated. Would personally move the status update to a separate job (e.g. post-build), instead of adding it here and changing the execution conditions (and semantic).